### PR TITLE
feat(sdk):  testing framework integration 2

### PR DESCRIPTION
**Pull request by afmika**
#### Motivation and context

Continuation of #566 , focused on prisma runtime.

#### Migration notes

N/A

### Checklist

- [x] The change come with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

**Review by zifeo — COMMENTED** [review]
Very nice! Waiting on @Natoandro or @Yohe-Am's review :)

**Review by Yohe-Am — DISMISSED** [review]

File: examples/deploy/deploy.mjs [hunk 1]
```diff
@@ -50,19 +62,50 @@ const tg = typegraph({
...
+      create: true,
+      reset: false
+    },
+    migrationDir: "prisma-migrations"
+  }
+};
+const baseUrl = "http://localhost:7890";
+const auth = new BasicAuth("admin", "password");
+
 tgDeploy(tg, {
```
Comment (Natoandro) [inline examples/deploy/deploy.mjs:84]: What happens with the `meta serialize` command?? :laughing:

**Review by Natoandro — COMMENTED** [review]
Nice work!

I also added some comments and questions on the previous PR (already merged)...

File: examples/deploy/deploy.mjs (hunk 1 above)
Comment (afmika) [reply examples/deploy/deploy.mjs:84]: We can always call `tg.serialize(config?)`

Comment (afmika) [reply examples/deploy/deploy.mjs:84]: Or maybe we can add `tgSerialize(tg, config?)` for consistency..

**Review by Yohe-Am — APPROVED** [review]

File: typegraph/python/typegraph/wit.py [hunk 2]
```diff
@@ -9,6 +9,11 @@
 from typegraph.gen.exports.runtimes import Runtimes
 from typegraph.gen.exports.utils import Utils
 from typegraph.host.host import HostImpl
+from typegraph.gen.exports.core import (
+    ArtifactResolutionConfig,  # noqa
+    MigrationConfig,  # noqa
```
Comment (zifeo) [inline typegraph/python/typegraph/wit.py:14]: why are those required? maybe add a short explanation after #noqa:

**Review by zifeo — APPROVED** [review]

File: typegraph/python/typegraph/wit.py (hunk 2 above)
Comment (afmika) [reply typegraph/python/typegraph/wit.py:14]: It's just a simple indirection to provide import similar to the node implementation.

**Review by Yohe-Am — APPROVED** [review]

**Review by Natoandro — APPROVED** [review]
